libs/search-widget/src/old-components/viewer/Viewer.svelte

```diff
@@ -87,6 +89,10 @@
   <div class="viewer-body" class:preview={$showPreview}>
     <div class="viewer-left">
       <InputViewer/>
+      {#if $notProcessed}
+        {$_('error.processing')}
+        <LoadingDots small />
```

Comment:
Do we really want loading dots here? I mean, will the viewer be automatically updated once processed is done? If not, you should not put a loader which imply you can wait for it, it will be there soon